src/Tribe/Views/V2/Views/Latest_Past_View.php

```diff
@@ -123,6 +123,9 @@ class Latest_Past_View extends View {
 		'latest-past/event/cost',
 		'latest-past/event/date-tag',
 		'latest-past/event/date/meta',
+
+		// Add-ons.
+		'components/filter-bar'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: b795cb6f033f93756bd9005990deeca2</sub>